app/examples/page.tsx

```
@@ -584,96 +598,134 @@ export default function ExamplesPage() {
+              )}
+
+              {/* Empty state */}
+              {filteredCardTypes.length === 0 && (
+                <div className="py-20 text-center">
+                  <p className="text-lg text-slate-600 dark:text-slate-400">
+                    No cards found matching your search.
+                  </p>
+                  <button
+                    onClick={handleClearFilters}
+                    className="mt-4 font-medium text-purple-600 hover:underline dark:text-purple-400"
+                  >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)